apps/account-api/src/services/handles.service.spec.ts

```diff
@@ -34,18 +35,18 @@ describe('HandlesService', () => {
     handlesService = moduleRef.get(HandlesService);
   });
   it('verifyHandleRequestSignature works with ethereum keys', async () => {
-    const { keyringPair, keypair } = createKeys('ethereum');
-    const ethAddr = keyringPair.address as HexString;
+    // @ts-ignore
+    const keyPair = createRandomEthereumKey();
```

Comment:
Upgrading @frequency/ethereum-utils broke this test. It now uses more functions from that package.